packages/next/src/client/components/error.tsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react'
 
-const styles: { [k: string]: React.CSSProperties } = {
+const styles: Record<PropertyKey, React.CSSProperties> = {
```

Comment:
```suggestion
const styles: Record<string, React.CSSProperties> = {
```

property key includes symbol and number too